cmd/crictl/image.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
nit: if you end up updating something, can you revert this change?

Reply:
actually, the boilerplate linter is failing because of this

Reply:
Strange, it is something that was changed when I used one of `go mod vendor` or `gofmt` but running either of them again after reverting the change does not mention it anymore.

Reply:
thanks!